client/src/app/pages/dashboard/maps/map.module.ts

```diff
@@ -25,6 +27,7 @@ import {MapListItemComponent} from './map-list/map-list-item/map-list-item.compo
 import {MapEditComponent} from './map-edit/map-edit.component';
 import { MapFavoritesComponent } from './map-favorites/map-favorites.component';
 import { MapListComponent } from './map-list/map-list.component';
+import 'hammerjs';
```

Comment:
What is this used for?

Reply:
Further investigation shows that the new gallery component needed it, but they recommend doing this import in `src/polyfills.js`

https://murhafsousli.github.io/ngx-gallery/#/getting-started/core

Reply:
I put it here only because of the fact that if preloadingStrategy would not be set to PreloadAllModules then angular would not load this script on every page refresh.

Reply:
Also for some reasons hammerjs is using up to 100% CPU just by moving the mouse around resulting in javascript event loop blocking and on mobile next button presses is sometimes not registered. Maybe other parts of the website are causing this because it's not the case in the example page. Hammerjs only powers optional gestures feature so I think it could be removed.